plugins/uefi/fu-plugin-uefi.c

```diff
@@ -620,6 +620,10 @@ fu_plugin_startup (FuPlugin *plugin, GError **error)
 	g_autofree gchar *nvram_total_str = NULL;
 	g_autoptr(GError) error_local = NULL;
 
+	/* don't let user's environment influence test suite failures */
+	if (g_getenv ("FWUPD_UEFI_TEST") != NULL)
+		return TRUE;
```

Comment:
So this was being set accidentally?

Reply:
Well we **intentionally** set that in the test suite for the dell plugin.  It's the only test suite that actually loads a real non-test plugin (IIRC).  So when that's set we want to make sure to ignore all startup checks.

Reply:
ahh, gotcha. LGTM then.